pom.xml

```diff
@@ -100,6 +100,12 @@
             <artifactId>spring-security-jwt</artifactId>
             <version>1.1.1.RELEASE</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.yannbriancon</groupId>
```

Comment:
do you need this dependency

Reply:
This dependency allows you to see where the n+1 problem occurs for hibernate. It will be useful if we continue to work on this project to correct all endpoint queries.

Reply:
cool